Makefile

```
@@ -50,6 +50,10 @@ build: third-party/my283
 debug: third-party/my283
 	cargo zigbuild --target=$(TARGET_TRIPLE).$(GLIBC_VERSION) --features=miyoo --bin=alliumd --bin=allium-launcher --bin=allium-menu --bin=activity-tracker --bin=screenshot --bin=say --bin=show --bin=show-hotkeys --bin=myctl
 
+.PHONY: strip-all
+strip-all:
+    find . -type f -exec file {}\; | awk -F ':' '/not stripped/ {print $1}' | while read f; do strip -s "$f"; done
```

third-party/my283/... should not be stripped. It is used for development only and not copied onto the device.

This syntax doesn't work on GNU find, use this:
```suggestion
	find dist static migrations -type f -exec file {} \; | awk -F ':' '/not stripped/ {print $1}' | while read f; do strip -s "$f"; done
```

And we are using tabs for this file, so keep the indentation the same.